include/zephyr/usb/usb_ch9.h

```diff
@@ -104,6 +104,17 @@ static inline bool usb_reqtype_is_to_device(const struct usb_setup_packet *setup
 #define USB_SREQ_SET_INTERFACE		0x0B
 #define USB_SREQ_SYNCH_FRAME		0x0C
 
+/** USB Hub Class Request Codes defined in spec. Table 11-16 */
```

Comment:
This and `USB_HCFS ` macros should rather go into the header `include/zephyr/usb/class/usb_hub.h`.